x-pack/plugins/maps/public/layers/sources/es_search_source/es_search_source.js

```diff
@@ -51,7 +51,7 @@ function getDocValueAndSourceFields(indexPattern, fieldNames) {
           lang: field.lang,
         },
       };
-    } else if (field.type !== ES_GEO_FIELD_TYPE.GEO_SHAPE && field.readFromDocValues) {
+    } else if (field.readFromDocValues) {
```

Comment:
Okay so the two cases here are:

* Old index pattern, `readFromDocValues` is false on geo_shape fields because it wasn't previously supported
* New index pattern, `readFromDocValues` is false because of the logic change in the index pattern service

Is that right?

Reply:
That is correct.

For old index pattern `readFromDocValues` was false because geo_shape did not support docvalues and aggregations.

For new index patterns `readFromDocValues` should be false even though geo_shape has docvalues now and can support aggregations.

Reply:
This line is removing a quick fix just merged (into master and 7.8) a few days ago. We decided a better fix is at the index pattern level.